ckanext/dia_theme/less/focus.less

```diff
@@ -24,6 +23,28 @@ header,
   }
 }
 
+header {
```

Comment:
I wonder if worth just adding a note here about why we basically duplicated the above to just remove button cos it could take a while to find the difference down the line!